libs/json/src/test/test_json_simple.c

```diff
@@ -24,6 +24,7 @@
 #include "json/json.h"
 
 static char *output = "{\"KeyBool\": true,\"KeyInt\": -1234,\"KeyUint\": 1353214,\"KeyString\": \"foobar\",\"KeyStringN\": \"foobarlong\",\"KeyIntArr\": [153,2532,-322]}";
+static char *output2 = "{\"KeySructArr\" : [{true, 2003, 'foobar'},{false, 2035, 'foobarst'}]}";
```

Comment:
Yes.
Sorry on that.


On 5/23/16, mkiiskila <notifications@github.com> wrote:
>> @@ -24,6 +24,7 @@
>>  #include "json/json.h"
>>
>>  static char *output = "{\"KeyBool\": true,\"KeyInt\": -1234,\"KeyUint\":
>> 1353214,\"KeyString\": \"foobar\",\"KeyStringN\":
>> \"foobarlong\",\"KeyIntArr\": [153,2532,-322]}";
>> +static char *output2 = "{\"KeySructArr\" : [{true, 2003,
>> 'foobar'},{false, 2035, 'foobarst'}]}";
>
> I assume you meant this to be:
> static char *output2 = "{\"KeySructArr\" : [{\"KeyBool\":true,
> \"KeyInt\":2003, \"KeyString\":\"foobar\"},"
>            "{\"KeyBool\":false, \"KeyInt\":2035,
> \"KeyString\":\"foobarst\"}]}";
>
> I think you need the key to go with every value.
> https://www.ietf.org/rfc/rfc4627.txt
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/apache/incubator-mynewt-core/pull/60/files/df3fe26cfc58b7f22fe9f8a48d4e5cf9a88be2e2#r64302111


-- 
"A Goal is a Dream with a Plan and a Dateline"
Do On to Others what you will like them to do on to you.'The Golden Rule'
Firefox Student Ambassadors Regional Lead Cameroon.
Software Developer,
Web Designer and Developer.
Music DJ . Artiste at Casky Black's Record